employees/common/exports.py

```diff
@@ -230,6 +231,26 @@ def _get_report_date_and_daily_hours(
         self._last_date = date
         return (report_date, daily_hours)
 
+    def _set_row_height(self, description: str) -> None:
+        # If the description has \ n or text exceeds 100 characters, it is automatically moved to the new line.
+
+        new_lines_in_description = description.count("\n")
+        splitted_description = description.split("\n")
+        rows_in_xlsx = 0
+
+        if len(splitted_description) == 1:
+            rows_in_xlsx = int(len(splitted_description[0]) / 100)
+        else:
+            for row in splitted_description:
+                rows_in_xlsx = rows_in_xlsx + int(len(row) / 100)
```

Comment:
You can simplify too `rows_in_xlsx += int(len(row) / 100)

Reply:
Yea... I was wondering why it doesn't work but I have tried `=+` ... thanks. Done